components/server/src/server.rs

```diff
@@ -991,6 +991,30 @@ where
             )),
         );
 
+        let mut debugger = DebuggerImpl::new(
+            engines.engines.clone(),
+            self.cfg_controller.as_ref().unwrap().clone(),
+            Some(storage),
+        );
+        debugger.set_kv_statistics(self.kv_statistics.clone());
+        debugger.set_raft_statistics(self.raft_statistics.clone());
+        // Debug service.
+        let debug_service = DebugService::new(
+            debugger,
+            server.get_debug_thread_pool().clone(),
+            engines.engine.raft_extension(),
+            self.pd_client.clone(),
+            self.region_info_accessor.clone(),
+        );
+        info!("start register debug service");
+        if server
+            .register_service(create_debug(debug_service))
+            .is_some()
+        {
+            fatal!("failed to register debug service");
+        }
+        info!("start register debug service");
```

Comment:
How about we initialize `debug_service` here and register it later in `register_services()`? This approach seems more appropriate for the process.

Reply:
done